docker-compose.yml

```diff
@@ -1,15 +1,23 @@
 version: '3.8'
 
 services:
+  godbo-lb:
+    image: nginx:mainline
+    container_name: godbo-nginx
+    ports:
+      - "8080:80"
```

Comment:
Maybe we can set an `.env` file to manage these environment varibles.

Reply:
Absolutely! This will become the next feature.

Reply:
OK. :D